pages/overview/concepts/pbt.md

```diff
@@ -1,5 +1,8 @@
 # What is a PBT?
-A PBT is a digital representation of a physical good as a non-fungible token, in order words a Physically Backed Token. Per [EIP-5791](https://eips.ethereum.org/EIPS/eip-5791) which defines the standard for PBTs, the standard was developed to solve the following problems:
+A PBT is a digital representation of a physical good as a non-fungible token, in order words a Physically Backed Token. 
+
+## Problems solved by PBTs
+Per [EIP-5791](https://eips.ethereum.org/EIPS/eip-5791) which defines the standard for PBTs, the standard was developed to solve the following problems:
 - The ownership of the physical item and the ownership of the NFT are decoupled.
```

Comment:
Wanted to make these key points more explicit/visible, as I think it's a important thing to understand